### DISCUSSION TITLE
Use DF without WooCommerce?

### DISCUSSION THREAD

**Discussion by e-orlov** [datafeedr/wordpress-plugins#37] [Help]
What are ways to use Datafeedr _without_ WooCommerce? 

In general, I imagine two ways:

1. Is it possible to use DF plugins "Products Sets" and "Comparison Sets" without WooCommerce?

2. Is it possible to use DF as data source (XML, CSV, JSON) to feed plugins like Content Egg?

**Comment by EricBusch** [comment]
Hi

The [Comparison Sets plugin](https://datafeedr.me/dfrcs) can be used without WooCommerce. You can just use shortcodes.

Yes, you can use the [Datafeedr API](https://datafeedr.github.io/datafeedr-api-docs/) as a data source though I don't know if you can integrate it with Content Egg.

Hope this helps!

If you have any other questions, please don't hesitate to ask.
